plotly/tools.py

```diff
@@ -269,12 +269,15 @@ def get_embed(file_owner_or_url, file_id=None, width="100%", height=525):
     Keyword arguments:
     file_id (default=None) -- an int or string that can be converted to int
                               if you're using a url, don't fill this in!
+    share_key (default=None) -- a string that enables private plots to be
+                                shared. Note, you must have admin privileges
+                                on the file for this to work
```

Comment:
if we keep this as share_key, we might want to explain how a user would *view* the share key for a plot.